libpromises/cf3parse_logic.h

```diff
@@ -87,4 +87,376 @@ static size_t CURRENT_PROMISER_LINE = 0;
 
 #define ParserDebug(...) LogDebug(LOG_MOD_PARSER, __VA_ARGS__)
 
+
+/*****************************************************************/
```

Comment:
IMO, it is cleaner to have this in a header file, since it should only ever be included (once), and never compiled separately. I suspect automake will try to compile it separately if I add it to sources list.